android/cli/commands/_build.js

```diff
@@ -4149,6 +4150,12 @@ AndroidBuilder.prototype.createUnsignedApk = function createUnsignedApk(next) {
 					&& !classRegExp.test(entry.name)
 					&& !trailingSlashRegExp.test(entry.entryName)
 				) {
+					// do not add duplicate entries
+					if (entryNames.indexOf(entry.entryName) > -1) {
+						this.logger.warn(__('Removing duplicate entry %s', entry.entryName.cyan));
```

Comment:
@cb1kenobi `warn` does not print any output (even with `-l trace`), switching to `info` does. Any ideas?